### PR TITLE
Update acceptance tests to use env variables and templates

### PR DESCRIPTION
**Pull request by schirevko**

### CODE REVIEW

File: vkcs/resource_vkcs_blockstorage_volume_snapshot_test.go [hunk 1]
```diff
@@ -122,27 +123,27 @@ func testAccBlockStorageSnapshotBasic() string {
...
     foo = "bar"
   }
 }
-`, testAccBlockStorageVolumeBasic)
+`, testAccRenderConfig(testAccBlockStorageVolumeBasic, testAccValues))
 }
 
 func testAccBlockStorageSnapshotUpdate() string {
 	return fmt.Sprintf(`
 %s
```
Comment (ftersin) [inline vkcs/resource_vkcs_blockstorage_volume_snapshot_test.go:138]: still not templatized

File: vkcs/resource_vkcs_blockstorage_volume_snapshot_test.go [hunk 2]
```diff
@@ -122,27 +123,27 @@ func testAccBlockStorageSnapshotBasic() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_blockstorage_volume_snapshot_test.go:123]: still not templatized

File: vkcs/resource_vkcs_db_database_test.go [hunk 3]
```diff
@@ -119,16 +120,16 @@ func testAccCheckDatabaseDatabaseDestroy(s *terraform.State) error {
 	return nil
 }
 
-var testAccDatabaseDatabaseBasic = fmt.Sprintf(`
-%s
-
-%s
+const testAccDatabaseDatabaseBasic = `
+{{.BaseNetwork}}
+
```
Comment (ftersin) [inline vkcs/resource_vkcs_db_database_test.go:125]: delete line

File: vkcs/resource_vkcs_keymanager_container_test.go [hunk 4]
```diff
@@ -190,17 +190,17 @@ resource "vkcs_keymanager_container" "container_1" {
```
Comment (ftersin) [inline vkcs/resource_vkcs_keymanager_container_test.go:190]: not templatized (see 5 lines above)

File: vkcs/resource_vkcs_keymanager_container_test.go [hunk 5]
```diff
@@ -216,17 +216,17 @@ resource "vkcs_keymanager_container" "container_1" {
```
Comment (ftersin) [inline vkcs/resource_vkcs_keymanager_container_test.go:216]: ditto

File: vkcs/resource_vkcs_keymanager_container_test.go [hunk 6]
```diff
@@ -252,17 +252,17 @@ resource "vkcs_keymanager_container" "container_1" {
```
Comment (ftersin) [inline vkcs/resource_vkcs_keymanager_container_test.go:252]: ditto

File: vkcs/resource_vkcs_keymanager_container_test.go [hunk 7]
```diff
@@ -282,17 +282,17 @@ resource "vkcs_keymanager_container" "container_1" {
```
Comment (ftersin) [inline vkcs/resource_vkcs_keymanager_container_test.go:282]: ditto

File: vkcs/resource_vkcs_lb_l7policy_test.go [hunk 8]
```diff
@@ -189,7 +189,7 @@ resource "vkcs_lb_l7policy" "l7policy_1" {
   action       = "REJECT"
```
Comment (ftersin) [inline vkcs/resource_vkcs_lb_l7policy_test.go:189]: ditto

File: vkcs/resource_vkcs_lb_l7policy_test.go [hunk 9]
```diff
@@ -203,7 +203,7 @@ resource "vkcs_lb_l7policy" "l7policy_1" {
   action       = "REDIRECT_TO_URL"
```
Comment (ftersin) [inline vkcs/resource_vkcs_lb_l7policy_test.go:203]: ditto

File: vkcs/resource_vkcs_lb_l7policy_test.go [hunk 10]
```diff
@@ -217,15 +217,15 @@ resource "vkcs_lb_pool" "pool_1" {
   name            = "pool_1"
```
Comment (ftersin) [inline vkcs/resource_vkcs_lb_l7policy_test.go:217]: ditto

File: vkcs/resource_vkcs_lb_l7policy_test.go [hunk 11]
```diff
@@ -238,14 +238,14 @@ resource "vkcs_lb_pool" "pool_1" {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_lb_l7policy_test.go:238]: ditto

File: vkcs/resource_vkcs_lb_l7rule_test.go [hunk 12]
```diff
@@ -259,7 +259,7 @@ func testAccCheckLbL7RuleConfigBasic() string {
 %s
```
Comment (ftersin) [inline vkcs/resource_vkcs_lb_l7rule_test.go:259]: ditto

File: vkcs/resource_vkcs_lb_l7rule_test.go [hunk 13]
```diff
@@ -272,7 +272,7 @@ func testAccCheckLbL7RuleConfigUpdate1() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_lb_l7rule_test.go:272]: ditto

File: vkcs/resource_vkcs_lb_l7rule_test.go [hunk 14]
```diff
@@ -286,7 +286,7 @@ func testAccCheckLbL7RuleConfigUpdate2() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_lb_l7rule_test.go:286]: ditto

File: vkcs/resource_vkcs_lb_l7rule_test.go [hunk 15]
```diff
@@ -300,7 +300,7 @@ func testAccCheckLbL7RuleConfigUpdate3() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_lb_l7rule_test.go:300]: ditto

File: vkcs/resource_vkcs_lb_l7rule_test.go [hunk 16]
```diff
@@ -314,7 +314,7 @@ func testAccCheckLbL7RuleConfigUpdate4() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_lb_l7rule_test.go:314]: ditto

File: vkcs/resource_vkcs_lb_l7rule_test.go [hunk 17]
```diff
@@ -327,7 +327,7 @@ func testAccCheckLbL7RuleConfigUpdate5() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_lb_l7rule_test.go:327]: ditto

File: vkcs/resource_vkcs_lb_l7rule_test.go [hunk 18]
```diff
@@ -341,7 +341,7 @@ func testAccCheckLbL7RuleConfigUpdate6() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_lb_l7rule_test.go:341]: ditto

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go [hunk 19]
```diff
@@ -307,10 +308,10 @@ func testAccNetworkingPortSecGroupAssociateManifestUpdate0() string {
 %s
```
Comment (ftersin) [inline vkcs/resource_vkcs_networking_port_secgroup_associate_test.go:308]: still not templatized

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go [hunk 20]
```diff
@@ -327,10 +328,10 @@ func testAccNetworkingPortSecGroupAssociateManifestUpdate2() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_networking_port_secgroup_associate_test.go:328]: ditto (and above)

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go [hunk 21]
```diff
@@ -341,12 +342,12 @@ func testAccNetworkingPortSecGroupAssociateManifestUpdate3() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_networking_port_secgroup_associate_test.go:342]: ditto

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go [hunk 22]
```diff
@@ -357,10 +358,10 @@ func testAccNetworkingPortSecGroupAssociateManifestUpdate4() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_networking_port_secgroup_associate_test.go:358]: ditto

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go [hunk 23]
```diff
@@ -371,11 +372,11 @@ func testAccNetworkingPortSecGroupAssociateManifestUpdate5() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_networking_port_secgroup_associate_test.go:372]: ditto

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go [hunk 24]
```diff
@@ -387,10 +388,10 @@ func testAccNetworkingPortSecGroupAssociateManifestUpdate6() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_networking_port_secgroup_associate_test.go:388]: ditto

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go [hunk 25]
```diff
@@ -401,10 +402,10 @@ func testAccNetworkingPortSecGroupAssociateManifestUpdate7() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_networking_port_secgroup_associate_test.go:402]: ditto

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go [hunk 26]
```diff
@@ -415,12 +416,12 @@ func testAccNetworkingPortSecGroupAssociateManifestUpdate8() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_networking_port_secgroup_associate_test.go:416]: ditto

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go [hunk 27]
```diff
@@ -431,10 +432,10 @@ func testAccNetworkingPortSecGroupAssociateManifestUpdate9() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_networking_port_secgroup_associate_test.go:432]: ditto

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go [hunk 28]
```diff
@@ -445,7 +446,7 @@ func testAccNetworkingPortSecGroupAssociateManifestUpdate10() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_networking_port_secgroup_associate_test.go:446]: ditto

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go [hunk 29]
```diff
@@ -457,10 +458,10 @@ func testAccNetworkingPortSecGroupAssociateManifestUpdate11() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_networking_port_secgroup_associate_test.go:458]: ditto

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go [hunk 30]
```diff
@@ -457,10 +458,10 @@ func testAccNetworkingPortSecGroupAssociateManifestUpdate11() string {
...
 resource "vkcs_networking_port_secgroup_associate" "port_1" {
-  port_id = "${data.vkcs_networking_port.hidden_port_1.id}"
+  port_id = data.vkcs_networking_port.hidden_port_1.id
   enforce = "false"
   security_group_ids = [
-    "${data.vkcs_networking_secgroup.default_2.id}",
+    data.vkcs_networking_secgroup.default_2.id,
   ]
 }
 `, testAccNetworkingPortSecGroupAssociate)
```
Comment (ftersin) [inline vkcs/resource_vkcs_networking_port_secgroup_associate_test.go:467]: ditto (below)

File: vkcs/resource_vkcs_sharedfilesystem_share_access_test.go [hunk 31]
```diff
@@ -170,14 +170,14 @@ func testAccSFSShareAccessConfigBasic() string {
 %s
```
Comment (ftersin) [inline vkcs/resource_vkcs_sharedfilesystem_share_access_test.go:170]: ditto

File: vkcs/resource_vkcs_sharedfilesystem_share_access_test.go [hunk 32]
```diff
@@ -192,14 +192,14 @@ func testAccSFSShareAccessConfigUpdate() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_sharedfilesystem_share_access_test.go:192]: ditto

File: vkcs/resource_vkcs_sharedfilesystem_share_test.go [hunk 33]
```diff
@@ -111,7 +111,7 @@ resource "vkcs_sharedfilesystem_share" "share_1" {
   share_proto      = "NFS"
```
Comment (ftersin) [inline vkcs/resource_vkcs_sharedfilesystem_share_test.go:111]: ditto (see 5 lines above)

File: vkcs/resource_vkcs_sharedfilesystem_share_test.go [hunk 34]
```diff
@@ -139,7 +139,7 @@ resource "vkcs_sharedfilesystem_share" "share_1" {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_sharedfilesystem_share_test.go:139]: ditto

File: vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go [hunk 35]
```diff
@@ -236,8 +236,8 @@ func testAccSFSShareNetworkConfigBasic() string {
 resource "vkcs_sharedfilesystem_sharenetwork" "sharenetwork_1" {
```
Comment (ftersin) [inline vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go:236]: ditto

File: vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go [hunk 36]
```diff
@@ -255,14 +255,14 @@ resource "vkcs_networking_subnet" "subnet_2" {
   name = "subnet_2"
```
Comment (ftersin) [inline vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go:255]: ditto

File: vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go [hunk 37]
```diff
@@ -297,10 +297,10 @@ func testAccSFSShareNetworkConfigSecService1() string {
 resource "vkcs_sharedfilesystem_sharenetwork" "sharenetwork_1" {
```
Comment (ftersin) [inline vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go:297]: ditto

File: vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go [hunk 38]
```diff
@@ -315,11 +315,11 @@ func testAccSFSShareNetworkConfigSecService2() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go:315]: ditto

File: vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go [hunk 39]
```diff
@@ -334,10 +334,10 @@ func testAccSFSShareNetworkConfigSecService3() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go:334]: ditto

File: vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go [hunk 40]
```diff
@@ -352,8 +352,8 @@ func testAccSFSShareNetworkConfigSecService4() string {
...
```
Comment (ftersin) [inline vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go:352]: ditto

File: vkcs/provider_test.go [hunk 41]
```diff
@@ -16,17 +18,34 @@ import (
...
 	osNetworkID        = os.Getenv("OS_NETWORK_ID")
 	osRegionName       = os.Getenv("OS_REGION_NAME")
-	osPoolName         = os.Getenv("OS_POOL_NAME")
 	osProjectID        = os.Getenv("OS_PROJECT_ID")
-	osExtGwID          = os.Getenv("OS_EXTGW_ID")
-	osPrivateDNSDomain = os.Getenv("OS_PRIVATE_DNS_DOMAIN")
+	osExtNetName       = os.Getenv("OS_EXT_NET_NAME")
 	osKeypairName      = os.Getenv("OS_KEYPAIR_NAME")
 	clusterTemplateID  = os.Getenv("CLUSTER_TEMPLATE_ID")
 	osSubnetworkID     = os.Getenv("OS_SUBNETWORK_ID")
```
Comment (ftersin) [inline vkcs/provider_test.go:30]: could we group kubernetes only related vars at the end of the list with a comment?

**Review by ftersin — CHANGES_REQUESTED** [review]

File: vkcs/data_source_vkcs_blockstorage_snapshot_test.go [hunk 42]
```diff
@@ -139,10 +42,12 @@ func testAccCheckBlockStorageSnapshotDataSourceID(n string) resource.TestCheckFu
 	}
 }
 
-func testAccBlockStorageSnapshotDataSourceBasic(snapshotName string) string {
+func testAccBlockStorageSnapshotDataSourceBasic() string {
 	return fmt.Sprintf(`
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_blockstorage_snapshot_test.go]: fixed

File: vkcs/data_source_vkcs_blockstorage_volume_test.go [hunk 43]
```diff
@@ -42,78 +26,12 @@ func TestAccBlockStorageVolumeDataSource_basic(t *testing.T) {
...
-		}
-
-		return nil
-	}
-}
-
-func testAccBlockStorageVolumeDataSourceBasic(volumeName string) string {
+func testAccBlockStorageVolumeDataSourceBasic() string {
 	return fmt.Sprintf(`
+%s
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_blockstorage_volume_test.go]: fixed

File: vkcs/data_source_vkcs_compute_quotaset_test.go [hunk 44]
```diff
@@ -10,6 +10,7 @@ import (
 
 func TestAccComputeQuotasetDataSource(t *testing.T) {
 	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
 		ProviderFactories: testAccProviders,
 		Steps: []resource.TestStep{
 			{
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_compute_quotaset_test.go:16]: fixed

File: vkcs/data_source_vkcs_db_backup_test.go [hunk 45]
```diff
@@ -46,6 +47,6 @@ var testAccDataSourceDatabaseBackupBasic = fmt.Sprintf(`
 %s
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_db_backup_test.go]: fixed

File: vkcs/data_source_vkcs_db_config_group_test.go [hunk 46]
```diff
@@ -47,6 +48,6 @@ var testAccDataSourceDatabaseConfigGroupBasic = fmt.Sprintf(`
 %s
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_db_config_group_test.go]: fixed

File: vkcs/data_source_vkcs_db_database_test.go [hunk 47]
```diff
@@ -46,6 +47,6 @@ var testAccDataSourceDatabaseDatabaseBasic = fmt.Sprintf(`
 %s
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_db_database_test.go]: fixed

File: vkcs/data_source_vkcs_db_instance_test.go [hunk 48]
```diff
@@ -48,4 +49,4 @@ var testAccDataSourceDatabaseInstanceBasic = fmt.Sprintf(`
 data "vkcs_db_instance" "basic" {
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_db_instance_test.go:49]: fixed

File: vkcs/data_source_vkcs_db_user_test.go [hunk 49]
```diff
@@ -46,6 +47,6 @@ var testAccDataSourceDatabaseUserBasic = fmt.Sprintf(`
 %s
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_db_user_test.go]: fixed

File: vkcs/data_source_vkcs_images_image_test.go [hunk 50]
```diff
@@ -124,7 +124,7 @@ func testAccImagesImageDataSourceBasic() string {
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_images_image_test.go:123]: fixed

File: vkcs/data_source_vkcs_keymanager_container_test.go [hunk 51]
```diff
@@ -57,7 +58,7 @@ func testAccKeyManagerContainerDataSourceBasic() string {
 %s
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_keymanager_container_test.go]: fixed

File: vkcs/data_source_vkcs_keymanager_container_test.go [hunk 52]
```diff
@@ -67,7 +68,7 @@ func testAccKeyManagerContainerDataSourceAcls() string {
...
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_keymanager_container_test.go]: fixed

File: vkcs/data_source_vkcs_networking_floatingip_test.go [hunk 53]
```diff
@@ -48,24 +49,20 @@ func testAccCheckNetworkingFloatingIPV2DataSourceID(n string) resource.TestCheck
...
+  pool = "{{.ExtNetName}}"
   description = "test fip"
 }
-`, testAccBaseExtNetwork)
-}
+`
 
 func testAccNetworkingFloatingIPV2DataSourceAddress() string {
 	return fmt.Sprintf(`
 %s
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_networking_floatingip_test.go]: fixed

File: vkcs/data_source_vkcs_networking_network_test.go [hunk 54]
```diff
@@ -164,8 +169,8 @@ func testAccNetworkingNetworkDataSourceBasic() string {
 %s
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_networking_network_test.go]: fixed

File: vkcs/data_source_vkcs_networking_network_test.go [hunk 55]
```diff
@@ -175,7 +180,7 @@ func testAccNetworkingNetworkDataSourceSubnet() string {
...
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_networking_network_test.go]: fixed

File: vkcs/data_source_vkcs_networking_network_test.go [hunk 56]
```diff
@@ -184,29 +189,25 @@ data "vkcs_networking_network" "network_1" {
...
-  name = "%s"
-}
-`, osPoolName)
+  name = "{{.ExtNetName}}"
 }
+`
 
 func testAccNetworkingNetworkDataSourceNetworkID() string {
 	return fmt.Sprintf(`
 %s
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_networking_network_test.go]: fixed

File: vkcs/data_source_vkcs_networking_router_test.go [hunk 57]
```diff
@@ -69,7 +70,7 @@ func testAccNetworkingRouterDataSourceName() string {
 %s
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_networking_router_test.go]: fixed

File: vkcs/data_source_vkcs_networking_secgroup_test.go [hunk 58]
```diff
@@ -100,7 +102,7 @@ func testAccNetworkingSecGroupDataSourceSecGroupID() string {
 %s
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_networking_secgroup_test.go]: fixed

File: vkcs/data_source_vkcs_networking_secgroup_test.go [hunk 59]
```diff
@@ -87,7 +89,7 @@ func testAccNetworkingSecGroupDataSourceBasic() string {
...
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_networking_secgroup_test.go]: fixed

File: vkcs/data_source_vkcs_networking_subnet_test.go [hunk 60]
```diff
@@ -143,7 +145,7 @@ func testAccNetworkingSubnetDataSourceBasic() string {
 %s
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_networking_subnet_test.go]: fixed

File: vkcs/data_source_vkcs_networking_subnet_test.go [hunk 61]
```diff
@@ -164,7 +166,7 @@ func testAccNetworkingSubnetDataSourceDhcpEnabled() string {
...
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_networking_subnet_test.go]: fixed

File: vkcs/data_source_vkcs_networking_subnet_test.go [hunk 62]
```diff
@@ -178,7 +180,7 @@ func testAccNetworkingSubnetDataSourceIPVersion() string {
...
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_networking_subnet_test.go]: fixed

File: vkcs/data_source_vkcs_networking_subnet_test.go [hunk 63]
```diff
@@ -189,7 +191,7 @@ func testAccNetworkingSubnetDataSourceGatewayIP() string {
...
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_networking_subnet_test.go]: fixed

File: vkcs/data_source_vkcs_sharedfilesystem_share_test.go [hunk 64]
```diff
@@ -52,13 +52,13 @@ resource "vkcs_sharedfilesystem_share" "share_1" {
   share_proto = "NFS"
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/data_source_vkcs_sharedfilesystem_share_test.go:51]: fixed

File: vkcs/resource_vkcs_blockstorage_volume_snapshot_test.go (hunk 1 above)
Comment (schirevko) [reply vkcs/resource_vkcs_blockstorage_volume_snapshot_test.go]: fixed

File: vkcs/resource_vkcs_blockstorage_volume_snapshot_test.go (hunk 2 above)
Comment (schirevko) [reply vkcs/resource_vkcs_blockstorage_volume_snapshot_test.go]: fixed

File: vkcs/resource_vkcs_keymanager_container_test.go (hunk 4 above)
Comment (schirevko) [reply vkcs/resource_vkcs_keymanager_container_test.go:189]: testAccDatabaseDatabaseBasic

Comment (schirevko) [reply vkcs/resource_vkcs_keymanager_container_test.go:189]: fixed

File: vkcs/resource_vkcs_keymanager_container_test.go (hunk 5 above)
Comment (schirevko) [reply vkcs/resource_vkcs_keymanager_container_test.go:213]: fixed

File: vkcs/resource_vkcs_keymanager_container_test.go (hunk 6 above)
Comment (schirevko) [reply vkcs/resource_vkcs_keymanager_container_test.go:247]: fixed

File: vkcs/resource_vkcs_lb_l7policy_test.go (hunk 8 above)
Comment (schirevko) [reply vkcs/resource_vkcs_lb_l7policy_test.go:188]: fixed

File: vkcs/resource_vkcs_lb_l7policy_test.go (hunk 9 above)
Comment (schirevko) [reply vkcs/resource_vkcs_lb_l7policy_test.go:200]: fixed

File: vkcs/resource_vkcs_lb_l7policy_test.go (hunk 11 above)
Comment (schirevko) [reply vkcs/resource_vkcs_lb_l7policy_test.go:231]: fixed

File: vkcs/resource_vkcs_lb_l7rule_test.go (hunk 12 above)
Comment (schirevko) [reply vkcs/resource_vkcs_lb_l7rule_test.go]: fixed

File: vkcs/resource_vkcs_lb_l7rule_test.go (hunk 14 above)
Comment (schirevko) [reply vkcs/resource_vkcs_lb_l7rule_test.go]: fixed

File: vkcs/resource_vkcs_lb_l7rule_test.go (hunk 16 above)
Comment (schirevko) [reply vkcs/resource_vkcs_lb_l7rule_test.go]: fixed

File: vkcs/resource_vkcs_lb_l7rule_test.go (hunk 15 above)
Comment (schirevko) [reply vkcs/resource_vkcs_lb_l7rule_test.go]: fixed

File: vkcs/resource_vkcs_lb_l7rule_test.go (hunk 17 above)
Comment (schirevko) [reply vkcs/resource_vkcs_lb_l7rule_test.go]: fixed

File: vkcs/resource_vkcs_lb_l7rule_test.go (hunk 18 above)
Comment (schirevko) [reply vkcs/resource_vkcs_lb_l7rule_test.go]: fixed

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go (hunk 19 above)
Comment (schirevko) [reply vkcs/resource_vkcs_networking_port_secgroup_associate_test.go]: fixed

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go (hunk 20 above)
Comment (schirevko) [reply vkcs/resource_vkcs_networking_port_secgroup_associate_test.go]: fixed

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go (hunk 21 above)
Comment (schirevko) [reply vkcs/resource_vkcs_networking_port_secgroup_associate_test.go]: fixed

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go (hunk 22 above)
Comment (schirevko) [reply vkcs/resource_vkcs_networking_port_secgroup_associate_test.go]: fixed

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go (hunk 23 above)
Comment (schirevko) [reply vkcs/resource_vkcs_networking_port_secgroup_associate_test.go]: fixed

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go (hunk 25 above)
Comment (schirevko) [reply vkcs/resource_vkcs_networking_port_secgroup_associate_test.go]: fixed

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go (hunk 26 above)
Comment (schirevko) [reply vkcs/resource_vkcs_networking_port_secgroup_associate_test.go]: fixed

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go (hunk 27 above)
Comment (schirevko) [reply vkcs/resource_vkcs_networking_port_secgroup_associate_test.go]: fixed

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go (hunk 28 above)
Comment (schirevko) [reply vkcs/resource_vkcs_networking_port_secgroup_associate_test.go]: fixed

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go (hunk 29 above)
Comment (schirevko) [reply vkcs/resource_vkcs_networking_port_secgroup_associate_test.go]: fixed

File: vkcs/resource_vkcs_networking_port_secgroup_associate_test.go (hunk 30 above)
Comment (schirevko) [reply vkcs/resource_vkcs_networking_port_secgroup_associate_test.go]: fixed

File: vkcs/resource_vkcs_sharedfilesystem_share_access_test.go (hunk 31 above)
Comment (schirevko) [reply vkcs/resource_vkcs_sharedfilesystem_share_access_test.go]: fixed

File: vkcs/resource_vkcs_sharedfilesystem_share_access_test.go (hunk 32 above)
Comment (schirevko) [reply vkcs/resource_vkcs_sharedfilesystem_share_access_test.go]: fixed

File: vkcs/resource_vkcs_sharedfilesystem_share_test.go (hunk 34 above)
Comment (schirevko) [reply vkcs/resource_vkcs_sharedfilesystem_share_test.go:134]: fixed

File: vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go (hunk 36 above)
Comment (schirevko) [reply vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go:252]: fixed

File: vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go (hunk 37 above)
Comment (schirevko) [reply vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go:292]: fixed

File: vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go (hunk 38 above)
Comment (schirevko) [reply vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go:308]: fixed

File: vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go (hunk 39 above)
Comment (schirevko) [reply vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go:325]: fixed

File: vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go (hunk 40 above)
Comment (schirevko) [reply vkcs/resource_vkcs_sharedfilesystem_sharenetwork_test.go:341]: fixed

File: vkcs/provider_test.go (hunk 41 above)
Comment (schirevko) [reply vkcs/provider_test.go]: Rearranged

**Review by ftersin — APPROVED** [review]

**Review by ftersin — APPROVED** [review]

**Pull request merged**